src/main/java/net/glowstone/entity/passive/GlowAbstractHorse.java

```diff
@@ -90,4 +101,20 @@ private int getHorseFlags() {
     public Set<Material> getBreedingFoods() {
         return BREEDING_FOODS;
     }
+
+    @Override
+    protected int computeGrowthAmount(Material material) {
+        int amount = 0;
+
+        // We need to be a baby and only tamed horses can be fed with hay block
+        if (!isAdult() && !(Material.HAY_BLOCK == material && !isTamed())) {
+            Integer mapResult = GROWING_FOODS.get(material);
+
+            if (mapResult != null) {
+                amount = Math.min(mapResult, Math.abs(getAge()));
```

Comment:
No need for that local variable. Just return at this line, and return `0` and the end of the method.